src/Raven.Server/Config/Categories/IndexingConfiguration.cs

```diff
@@ -123,6 +124,27 @@ public virtual bool RunInMemory
         [ConfigurationEntry("Indexing.CleanupIntervalInMin", ConfigurationEntryScope.ServerWideOnly)]
         public TimeSetting CleanupInterval { get; set; }
 
+        [Description("Maximum amount of scratch space that we allow to use for an indexing batch")]
+        [DefaultValue(null)]
+        [SizeUnit(SizeUnit.Megabytes)]
+        [IndexUpdateType(IndexUpdateType.Refresh)]
+        [ConfigurationEntry("Indexing.ScratchSpaceLimitInMb", ConfigurationEntryScope.ServerWideOrPerDatabase)]
```

Comment:
Yes. It can be defined per index as setting any other index specific configuration using `IndexConfiguration` class in client API via `IndexDefinition.Configuration`. It's also exposed in the studio.